app-i18n/libhangul/libhangul-0.1.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="Library to support hangul input method logic"
+DESCRIPTION="A library to support hangul input method logic, hanja dictionary and small hangul \
+character classification."
+HOMEPAGE="https://github.com/libhangul/libhangul"
+COPYRIGHT="2004-2021 Hwangjin Choe"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/libhangul/libhangul/archive/refs/tags/libhangul-0.1.0.zip"
+CHECKSUM_SHA256="e90fb326167f8d54ba637bf5d1d55cbe9956f6b03a7ae9cb0786037b4a2a487b"
+SOURCE_DIR="libhangul-libhangul-$portVersion"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="1.0.0"
+libVersionCompat="$libVersion compat >= 1"
+
+PROVIDES="
+	libhangul$secondaryArchSuffix = $portVersion
+	cmd:hangul
+	lib:libhangul$secondaryArchSuffix = $libVersionCompat
```

Comment:
The KoreanIM uses a makefile that checks for libhangul (not for libhangul_x86), so I would suggest dropping $secondaryArchSuffix all together (here and through the whole recipe):
```diff
-	libhangul$secondaryArchSuffix = $portVersion
+	libhangul = $portVersion
	cmd:hangul
-	lib:libhangul$secondaryArchSuffix = $libVersionCompat
+	lib:libhangul = $libVersionCompat
```